extensions/amp-form/0.1/validation-bubble.js

```diff
@@ -103,7 +135,13 @@ function measureTargetElement(state) {
  * @private
  */
 function showBubbleElement(state) {
-  state.bubbleElement.textContent = state.message;
+  removeChildren(state.bubbleElement);
+  const messageDiv = state.bubbleElement.ownerDocument.createElement('div');
+  messageDiv.id = state.bubbleElement.getAttribute('aria-labeledby');
+  messageDiv.textContent = state.message;
+  state.bubbleElement.setAttribute('role', 'alert');
```

Comment:
It is a recommendation by WCAG working group, it allows the user to  read the validation message again easily next time they focus on the input to fix it as the input would be announced by its label first followed by the error message.